source/developer/interactive-dialogs.rst

```diff
@@ -274,6 +274,12 @@ Moreover, Mattermost also allows the integration itself to perform input validat
 
   {"errors": {"num_between_0_and_10": "Enter a number between 0 and 10."}}
 
+The integration may also return a generic error message to the user that is not attached to a specific field. This can be done by responding to the dialog submission request with a JSON body containing an `error` field.  The `error` field should contain a string with the error message to display to the user.  For example, if a server-side error occurs, you can return a message explaining it:
+
```

Comment:
Would you please add a note about the required server version for this? Something like:
```suggestion

	This was added in Mattermost 5.16

```

Reply:
Done.  I added it just after the JSON example.